scripts/tasks/fingerprint.js

```diff
@@ -1,12 +1,13 @@
 module.exports = function (gulp, plugins, consts) {
     return function () {
         var fonts = plugins.path.join(consts.SRC, 'fonts', '*');
-        var images = plugins.path.join(consts.SRC, 'images', '*');
-        var logos = plugins.path.join(consts.SRC, 'images', 'logos', '*');
+        var images = plugins.path.join(consts.SRC, 'images', '**', '*');
+        var icons = plugins.path.join(consts.SRC, 'images', 'icons', '*');
+        var subjects = plugins.path.join(consts.SRC, 'images', 'subjects', '*');
 
         // by default, gulp would pick `assets/css` as the base,
         // so we need to set it explicitly:
-        return gulp.src([fonts, images, logos], {base: './src'})
+        return gulp.src([fonts, images, '!' + icons, '!' + subjects], {base: './src'})
```

Comment:
We exclude from fingerprinting just icons and subject images as they are build into the sprite and not used as single units